SmartCharger/Controllers/LoginController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SmartCharger.Business.DTOs;
+using SmartCharger.Business.Interfaces;
+
+namespace SmartCharger.Controllers
+{
+    [Route("api/")]
+    [ApiController]
+    public class LoginController : ControllerBase
+    {
+        private readonly ILoginService _loginService;
+
+        public LoginController(ILoginService loginService)
+        {
+            _loginService = loginService;
+        }
+
+        [HttpPost("login")]
+        public async Task<IActionResult> Login([FromBody] LoginDTO loginDTO)
+        {
+            LoginResponseDTO registrationResult = await _loginService.LoginAsync(loginDTO);
```

Comment:
I think it would be better to rename "registrationResult" to something like "loginResult".

Reply:
Thanks for spotting this mistake. I renamed variable to loginResult.